src/main/resources/ascent-dashboard.yml

```diff
@@ -12,6 +12,10 @@ security:
   user:
       name: ${ascent.dashboard.username:admin}
       password: ${ascent.dashboard.password:default}
+  gateway:
+      user:
+          name: ${ascent.gateway.username}
```

Comment:
Did the developers want to provide a default value for these when they are running locally?